src/languages/es.js

```diff
@@ -485,6 +485,7 @@ export default {
         error: {
             invalidFormatEmailLogin: 'El email introducido no es válido. Corrígelo e inténtalo de nuevo.',
         },
+        cannotGetAccountDetails: 'No se pueden obtener los detalles de la cuenta. Por favor inténtalo de nuevo.',
```

Comment:
Why was this changed to plural? It sounds odd like this.

Reply:
That was just a placeholder I had from Google Translate